src/features/search/index.tsx

```diff
@@ -44,20 +45,14 @@ const DateRowLayoutContainer = styled.div`
 
 type Search = (query: string) => void;
 
-type StockListItem = {
-    symbol: string,
-    name: string
-}
-
-
 const socket = socketService.get();
 
 export const Search: FC = () => {
 
     const dispatch = useDispatch();
 
     const [query, setQuery] = useState<string>('Apple Inc (AAPL)');
-    const [stockList, setStockList] = useState<StockListItem[]>([])
+    // const [stockList, setStockList] = useState<StockListItem[]>([])
```

Comment:
remove the comment please

Reply:
must of missed these after testing to make sure it all worked.

both have been fixed and pushed.